### DISCUSSION TITLE
RFID-RC522 ESP32 über Pin mit Spannung versorgen

### DISCUSSION THREAD

**Discussion by andistg** [biologist79/ESPuino#79] [Q&A]
Hallo Zusammen, 

ich habe eine Art Dash-Button mit einem RC-522 gebaut (Damit nicht jeder drücken kann, sondern nur mit den entsprechenden Karten der Button freigeschaltet werden kann). Soweit funktioniert das alles auch gut. 

Da ich das ganze mit einem Akku betreiben möchte, versuche ich gerade den Reader über über einen Pin (ESP32 Wroom 32)ein- und auszuschalten. Auch das funktioniert soweit gut, allerdings erkennt der Reader dann nicht mehr die RFID-Karten. Mit direkter Versorgung zum Akku klappt es aber. Hat jemand eine Idee, woran das liegen kann? 

Danke und beste Grüße. Andreas

**Comment by gmbo** [comment]
Der Reader zieht gute 20mA Strom bei der Initialisierung und beim Lesen mehr.
das wird der Ausgang nicht liefern.
Aus dem Grund wird dann oft ein FET benutzt um zu schalten.
Ob es reicht die LED wegzulassen weiß ich aber auch nicht.

**Comment by andistg** [comment]
Ok! Daran habe ich nicht gedacht. Ich teste mal. Die LED habe ich schon raus gelötet! Danke 🙏

**Comment by biologist79** [comment]
Also ich habe das in der Anfangszeit gemacht, dass ich den RC522 direkt über einen GPIO geschaltet habe. Möglicherweise mag das nicht schlau gewesen sein (deswegen jetzt auch FET), aber grundsätzlich funktioniert hat es (bei mir). Aber ja, teste es mal mit einem FET.
Hinweis an dieser Stelle: Es gab Leute, die mit dem IRF530N Probleme hatten. Möglicherweise war die Vgs mit 2 bis 4 V zu hoch. Ich habe daher kürzlich auf einen IRL3103 gewechselt. Also nur für den Fall, dass du doch auf FET-Ansteuerung gehen willst :-)

**Reply by andistg** [reply]
Super! Vielen Dank für die schnelle Hilfe 😊! Material ist bestellt....😊
